docs/theory-of-operation.md

```diff
@@ -206,3 +206,25 @@ chip. To enter hidden menu type `shift + z` in payload menu.
 
   Watch [video](https://asciinema.org/a/244774) showing how to enable/disable
   SHAx banks in TPM menu.
+
+- **Setting watchdog timeout**
+
+  To `Enable` watchdog, type `i` in payload menu. You will be prompted to
+  specify the timeout after which the platform should reset.
+
+  By default the timeout is set to 0 seconds (disabled state). To enable
+  watchdog, enter setup menu and toggle watchdog option.
+
+  > WARNING: do not set short timeouts! It may lead to a reset loop and brick
+  Your platform. Please take into consideration that platform boot time and OS
+  boot time also counts to the overall timeout time, so set at least few minutes
+  timeout to still be able to enter setup menu to disable the watchdog!
```

Comment:
The information that the sortbootorder payload won't let setting a value lower than 60s since v4.14.0.4 version, would be great to put here.

Reply:
Added this information